engine/src/test/java/com/datatorrent/stram/engine/AutoMetricTest.java

```diff
@@ -266,7 +265,6 @@ public void testMetrics() throws Exception
   }
 
   @Test
-  @Ignore
   public void testMetricsAggregations() throws Exception
```

Comment:
I don't understand how your modified test verifies the code changes you have made. Only the number of partitions have been changed from 2 to 4 but that doesn't verify the partial metrics change you have made.

Reply:
Previously, this test is intermediately failed due to the partial metrics. I think that the chance of failure this test would be more if we increased the number of partitions.